recipes/seqeval/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "seqeval" %}
+{% set version = "0.0.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9c7764b88b6d1430ab59de63efddf19495fca82159920fd11e362a405f8fc378
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools_scm
+  run:
+    - keras >=2.2.4
+    - {{ pin_compatible('numpy') }}
```

Comment:
you only need pin compatible if you are compili and linking against numpy. A run-only dep can just be 'numpy'